source/components/ui/msm-modal/msm-modal.service.js

```diff
@@ -161,33 +161,23 @@
      *
      * @description
      *     Display a modal selection dialog.
-     * @param {string} title
-     *     The modal's title.
-     * @param {string} text
-     *     The modal's text.
      * @param {array} options
      *     The modal's selection options.
-     * @param {string=} size
-     *     The modal's size.
-     * @param {string=Ok} closeTitle
-     *     The label of the modal's confirm button.
-     * @param {string=Cancel} dismissTitle
-     *     The label of the modal's cancel button.
      */
-    function select(title, text, options, size, closeTitle, dismissTitle) {
-      return open({ options: options }, function ($modalInstance, options) {
+    function select(options) {
```

Comment:
The benefit of it is that it's much clearer for a parameter list as we don't have to count through our 5 or more parameters.